php/Terminus/Commands/OrganizationsCommand.php

```diff
@@ -227,28 +239,72 @@ public function team($args, $assoc_args) {
     $org_info  = $org->get('organization');
     $org_model = new Organization($org_info);
 
-    $memberships = $org->user_memberships->all();
-    $data        = array();
-    foreach ($memberships as $membership) {
-      $member = $membership->get('user');
+    switch ($action) {
+      case 'add-member':
+        $email = $this->input()->string(
+          [
+            'args'    => $assoc_args,
+            'key'     => 'member',
+            'message' => 'What is the email address of the user to be added?',
+          ]
+        );
+        $role     = $this->input()->role(['args' => $assoc_args]);
```

Comment:
We need to add some Change Management logic here. If the Org does not have CM, the only valid roles are `unprivileged` and `admin`. If the org does have CM, the roles are `unprivileged`, `developer`, `team_member`, and `admin`.

We probably need separate input role helpers for Orgs and Site Teams.